Orange/widgets/regression/owlinearregression.py

```diff
@@ -101,7 +103,20 @@ def _alpha_changed(self):
 
     def commit(self):
         alpha = self.alphas[self.alpha_index]
-        args = {"preprocessors": self.preprocessors}
+        preprocessors = self.preprocessors
+        if self.data is not None and np.isnan(self.data.Y).any():
+            self.warning(0, "Missing values in class variable(s)")
```

Comment:
Instances with missing values of dependent variable are removed from the data.

We should not say "class" on regression data.